docker-compose.yml

```diff
@@ -29,6 +29,8 @@ x-aliases:
         - PHP_IDE_CONFIG=serverName=docker
       cap_add:
         - SYS_PTRACE
+      # Privileged is requires to run some pcntl tests locally.
```

Comment:
I added it here. The hope is that after I pass over it for 100 times, I will remember to enable it when I need to run the whole pcntl php language tests suite :D (which runs in CI)